resources/css/app.css

```
@@ -202,4 +267,8 @@ div.thread-card:nth-last-child(2) {
     nav li.active {
         @apply border-b-2 border-green-primary;
     }
+}
+
+[v-cloak] {
+  display: none;
```

Always use 4 spaces for indenting everywhere.

```suggestion
    display: none;
```